stable/busybox/templates/deployment.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

Comment:
👍  to standard labels

re: not setting resources by default—while we don't call for this explicitly in our technical requirements, I can see the merit. Wouldn't consider it blocking for this PR.